guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -55,39 +55,42 @@ mysqldb:
             importer:
                 privileges:
                     main: rw
-
 ```
 
-In the preceding example, the endpoint (that is, user) `reporter` has `ro` privileges to the `main` database and endpoint `importer` has `rw` access to the `main` database. This means that:
+In the preceding example, the endpoint (user) `reporter` has read-only privileges to the `main` database and endpoint `importer` has read-write access to the `main` database. This means that:
 
 -  The `admin` user has full control of the database.
 -  The `repoter` user has SELECT privileges only.
 -  The `importer` user has SELECT, INSERT, UPDATE, and DELETE privileges.
 
-## Add MySQL in services.yaml and .magento.app.yaml {#settings}
+## Enable MySQL
 
-To enable MySQL, add the following code with your installed version and allocated disk space in MB to `.magento/services.yaml`.
+1.  Add the required name, type, and disk value (in MB) to the `.magento/services.yaml` file.
 
-```yaml
-mysql:
-    type: mysql:10.0
-    disk: 2048
-```
+  ```yaml
+  mysql:
+      type: mysql:10.2
+      disk: 2048
+  ```
 
-To configure the relationships for the environment variable, set a relationship in your `.magento.app.yaml` in the Git branch. For example:
+1.  Configure the relationships in the `.magento.app.yaml` file.
 
-```yaml
-relationships:
-    database: "mydatabase:mysql"
-```
+  ```yaml
+  relationships:
+      database: "mydatabase:mysql"
```

Comment:
where this `mydatabase` came from? I see in the example above `mysql` name in services.yaml

Reply:
That could have been leftover from some old stuff. Sorry to not catch that. (Double-checking my own cloud in case that nonsense came from me.) 😟